src/components/mediainfo/mediainfo.js

```diff
@@ -261,7 +261,13 @@ import 'emby-button';
 
                 minutes = minutes || 1;
 
-                miscInfo.push(`${Math.round(minutes)} mins`);
+                if (item.UserData?.PlaybackPositionTicks) {
+                    let remainingMinutes = (item.RunTimeTicks - item.UserData.PlaybackPositionTicks) / 600000000;
+                    remainingMinutes = remainingMinutes || 1;
+                    miscInfo.push(`${Math.round(minutes)} mins (${Math.round(remainingMinutes)} remaining)`);
```

Comment:
This should be translated.

Reply:
Added translation in the new commit.  Wasn't sure what the process was for adding new strings to be translated, so I just tacked it on at the end of the en-us.json file.  Let me know if there's something else I should have done.